script.js

```diff
@@ -0,0 +1,17 @@
+let total = 0;
+
+let dogs = {
```

Comment:
Awesome! Love the object for info.

One suggestion for formatting it's nice to lay out objects like this.

```

let dogs = {
  'Luna': {
    'breed' : 'Yellow Lab', 
    'price': 100.00
  },
  'George': {
    'breed' : 'Pug', 
    'price': 200.00
  },
  'Lucy': {
    'breed' : 'Golden Retriever', 
    'price': 300.00
  }
}
```
It makes it a little easier to see the key value pairs